viewer/viewer/templates/viewer/help.html

```diff
@@ -1,122 +1,146 @@
-  </p>
-  <p>
-    <a href="{% url 'index' %}?search_type=links&q=none">Search "none"</a>
-  </p>
-  <p>
-    You're looking for incorrectly formatted document links.
-  </p>
-  <p>
-    <a href="{% url 'index' %}?search_type=links&q={{ 'www.consumerfinance.gov/documents' | urlencode }}">Search "www.consumerfinance.gov/documents"</a>
-  </p>
+	<h2>Links</h2>
+	<p>
+		Find all instances of a string in the URLs of links listed on a page,
+		excluding the header and footer.
+	</p>
+	<p class="u-example_text">Examples</p>
+
+	<ul class="m-list">
+		<li class="m-list_item">
+			You're looking for all instances of external links to TBD.
+			<a href="{% url 'index' %}?search_type=links&q=TBD">Search "TBD"</a>
+		</li>
+		<li class="m-list_item">
+			You want to find and remediate accidental links to the content
+			serve.
+			<a
+				href="{% url 'index' %}?search_type=links&q=content.consumerfinance.gov">
+				Search "content.consumerfinance.gov"
+			</a>
+		</li>
+		<li class="m-list_item">
+			You want to find published cf.gov pages with broken links.
+			<a href="{% url 'index' %}?search_type=links&q=none">
+				Search "none"
+			</a>
+		</li>
+		<li class="m-list_item">
+			You're looking for incorrectly formatted document links.
+			<a
+				href="{% url 'index' %}?search_type=links&q={{ 'www.consumerfinance.gov/documents' | urlencode }}">
+				Search "www.consumerfinance.gov/documents"
+			</a>
+		</li>
+	</ul>
 </div>
 
 <div class="block">
-  <h3>Full text</h3>
-  <p>
-    Find all instances of a term or exact phrase in the full text of the
-    &lt;body&gt; of a page, excluding the header and footer.
-  </p>
-  <p>
-    You're looking for specific phrasing or language that might need to be
-    updated because of a policy change.
-  </p>
-  <p>
-    <a href="{% url 'index' %}?search_type=text&q={{ 'medical debt' | urlencode }}">Search "medical debt"</a>
-  </p>
+	<h2>Full text</h2>
+	<p>
+		Find all instances of a term or exact phrase in the full text of the
+		&lt;body&gt; of a page, excluding the header and footer.
+	</p>
+	<p class="u-example_text">Examples</p>
+
+	<ul class="m-list">
+		<li class="m-list_item">
+			You're looking for specific phrasing or language that might need to
+			be updated because of a policy change.
+			<a
+				href="{% url 'index' %}?search_type=text&q={{ 'medical debt' | urlencode }}">
+				Search "medical debt"
+			</a>
+		</li>
+	</ul>
 </div>
 
 <div class="block">
-  <h3>HTML</h3>
-  <p>
-    You want to find and remediate manually-added line breaks.
-  </p>
-  <p>
-    <a href="{% url 'index' %}?search_type=html&q={{ '<br>' | urlencode }}">Search "&lt;br&gt;"</a>
-  </p>
+	<h2>HTML</h2>
+	<p>Search HTML</p>
+	<p class="u-example_text">Examples</p>
+
+	<ul class="m-list">
+		<li class="m-list_item">
+			You want to find and remediate manually-added line breaks.
+			<a
+				href="{% url 'index' %}?search_type=html&q={{ '<br>' | urlencode }}">
+				Search "&lt;br&gt;"
+			</a>
+		</li>
+	</ul>
```

Comment:
Can we add another example here for searching the HTML for inclusion of a specific image? Something like

```html
You want to find pages that include a specific image filename.

<a href="{% url 'index' %}?search_type=html&q=consumer_tools_hero">
        Search "consumer_tools_hero"
</a>
```